roles/core/tasks/install.yml

```diff
@@ -39,9 +39,15 @@
     dest: "/tmp/sdcore-5g-values.yaml"
   when: inventory_hostname in groups['master_nodes']
 
+- name: copy local_charts "{{ core.helm.chart_ref }}" to /tmp/sdcore-helm-charts
+  copy:
+    src: "{{ core.helm.chart_ref }}/"
```

Comment:
This is not correct because it needs to copy the charts one level up. That is, if the local charts umbrella is located at`/home/user/sdcore-helm-charts/sdcore-helm-charts`, then, what needs to be copied is `/home/user/sdcore-helm-charts`

FYI, the Helm Charts include the following directories (Charts)., where one of them (`sdcore-helm-charts`) are the Umbrella Helm Charts and this one calls (points) to the other Charts and this is why we need to copy the parent directory including the other Charts
```
$ ls sdcore-helm-charts/
5g-control-plane
5g-test-apps
5g-ran-sim
bess-upf
omec-control-plane
omec-sub-provision
router
sdcore-helm-charts
...
```

Reply:
I got it, I thought we were copying and deploying 1 by 1. For example, for 5GC we could only use `5g-control-plane` charts.

I will make corresponding changes and update.

Reply:
No, the deployment is done per-"Umbrella" chart and the "umbrella" chart uses other charts that are at the same level as the "umbrella"